homeassistant/components/elkm1/__init__.py

```diff
@@ -221,6 +231,23 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     )
     elk.connect()
 
+    def _element_changed(element, changeset):
+        keypress = changeset.get("last_keypress")
+        if keypress is None:
+            return
+
+        hass.bus.async_fire(
+            EVENT_ELKM1_KEYPAD_KEY_PRESSED,
+            {
+                ATTR_KEYPAD_ID: element.index + 1,
+                ATTR_KEY_NAME: keypress[0],
+                ATTR_KEY: keypress[1],
+            },
+        )
+
+    for keypad in elk.keypads:
```

Comment:
```suggestion
    for keypad in elk.keypads: # pylint: disable=no-member
```
Since https://github.com/gwww/elkm1/blob/main/elkm1_lib/elk.py#L70 is creating these dynamically.